File: token-validation-spring-demo/src/main/java/no/nav/security/token/support/demo/spring/DemoApplication.java
package no.nav.security.token.support.demo.spring;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class DemoApplication {

	public static void main(String[] args) {
		SpringApplication app = new SpringApplication(DemoApplication.class);
		app.setAdditionalProfiles("dev");
		app.run(args);
	}
}

File: token-validation-spring-demo/src/main/kotlin/no/nav/security/token/support/demo/spring/DemoApplication.kt
package no.nav.security.token.support.demo.spring

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class DemoApplication {

    fun main(args : Array<String>) {
        val app = SpringApplication(DemoApplication::class.java)
        app.run(*args)
    }
}